flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobManagerServices.java

```diff
@@ -116,8 +116,15 @@ public static JobManagerServices fromConfiguration(
 		final String classLoaderResolveOrder =
 			config.getString(CoreOptions.CLASSLOADER_RESOLVE_ORDER);
 
+		final String alwaysParentFirstLoaderString =
```

Comment:
I think this could use some exception handling, for example when the parameter is null, or empty, or does not split.

Reply:
The parameter cannot be `null`, since there is a default value. If it is empty, won't `split` simply return an empty array?

I made a comment about this below, i.e. where we should handle parsing of config options.